about/index.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: About the Club
+---
+
+# About
+
+The University of Waterloo Amateur Radio Club is a club at the
```

Comment:
I'm curious (@mtrberzi might be able to answer this one): are we considered a club-club or a team? How we're recognized by the university is actually pretty unique, and club usually implies Society or FEDS club. Does EngSoc recognize UWARC? (They don't normally recognize clubs because they represent programs and teams AFAIK.)